web/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
So we need this file in 2 places, or can we just keep one in source, ie. `services/openshift-service-impl/src/main/resources/META-INF/jboss-deployment-structure.xml` ?